components/app/BreadCrumbs.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+  <div class="breadcrumb-container">
+    <a-breadcrumb-item>
+      <a href="/">
+        <a-icon type="home" />
+      </a>
+    </a-breadcrumb-item>
+    <a-breadcrumb>
+      <a-breadcrumb-item v-for="crumb in crumbs" :key="crumb.path">
+        <a :href="crumb.to">{{ crumb.text }}</a>
+      </a-breadcrumb-item>
+    </a-breadcrumb>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'BreadCrumbs',
+  computed: {
+    crumbs() {
+      const pathArray = this.$route.path.split('/');
+      const breadcrumbs = pathArray
+        .slice(1)
+        .reduce((breadcrumbArray, path, idx) => {
+          return [
+            ...breadcrumbArray,
+            {
+              path,
+              to: breadcrumbArray[idx - 1]
+                ? '/' + breadcrumbArray[idx - 1].path + '/' + path
+                : '/' + path,
+              text: this.capitalizeFirstLetter(
+                this.$route.matched[idx]?.meta.breadCrumb || path
+              )
+            }
+          ];
+        }, []);
+      return breadcrumbs || [];
+    }
+  },
+  methods: {
+    capitalizeFirstLetter(text) {
```

Comment:
I'd rather use a CSS class with `text-transform: capitalize`, because I think it's cleaner, but just a comment.